bi_sql_editor/README.rst

```diff
@@ -0,0 +1,185 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```

Comment:
Use the .png image and the https for the links. See the updated template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst